### PR TITLE
tools: Apply M16 rule at clang-format file

### PR DESCRIPTION
**Pull request by junmin-kim**
[M16] Open braces for enum, union and struct go on the same line

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [4e670c8c133d88c5ea55915f411ef27ff4382bd8] -

**Comment by seinfra** [comment]
Target : [4e670c8c133d88c5ea55915f411ef27ff4382bd8] -

**Comment by seinfra** [comment]
Target : [628666eff80e49af0e84efa2084987e37a6ad81c] - Code Rule Check Result:
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:170: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:170: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:170: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:170: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:190: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:190: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:190: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/MediaRecorder.h:170: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaRecorder.h:170: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:170: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaRecorder.h:170: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/MediaRecorder.h:181: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:181: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaRecorder.h:181: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)

File: framework/include/media/MediaPlayer.h [hunk 1]
```diff
@@ -29,8 +29,7 @@ namespace media {
  * @details @b #include <media/MediaPlayer.h>
  * @since TizenRT v2.0 PRE
  */
-typedef enum player_state_e
-{
+typedef enum player_state_e {
```
Comment (sunghan-chang) [inline framework/include/media/MediaPlayer.h]: These changes are not ```tools: Apply M16 rule at clang-format file```. Please split them

**Comment by seinfra** [comment]
Target : [49b33c34b0148a3daa98b95410bcd6b561165354] - Code Rule Check Result:
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:162: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:170: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:170: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:170: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:170: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:190: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:190: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:190: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:162: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/MediaRecorder.h:170: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaRecorder.h:170: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:170: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaRecorder.h:170: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/include/media/MediaRecorder.h:181: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaRecorder.h:181: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaRecorder.h:181: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)

**Comment by seinfra** [comment]
Target : [49b33c34b0148a3daa98b95410bcd6b561165354] - Code Rule Check (C++) OK.

**Comment by sunghan-chang** [comment]
@yaniardiyanto Thank you for giving opinion. But it looks incorrect information on Tizen Coding Rule site. Actually this PR is correct. I am checking site and rule with our engineer who is in charge of coding rule.

**Pull request merged**